styles/main.scss

```diff
@@ -30,4 +31,84 @@ $font-sizes: (
 .footer-icon:hover {
   color:#4592CF
 }
-  
\ No newline at end of file
+
+// About Page Classes
```

Comment:
Is there some way we can nest these? Would look cleaner (to me, at least) if it was sitting under some kinda `about` section

Reply:
All the CSS for the about page is now under the #about id, which is the outer container that contains all of the About Page's components.